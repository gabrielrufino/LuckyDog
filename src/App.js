import 'react-native-gesture-handler';

import React, {useEffect} from 'react';
import SplashScreen from 'react-native-splash-screen';
import {StatusBar} from 'react-native';

import Navigator from './Navigator';

export default function App() {
  useEffect(() => {
    SplashScreen.hide();
  }, []);

  return (
    <>
      <StatusBar backgroundColor="#C8AB83" />

      <Navigator />
    </>
  );
}
