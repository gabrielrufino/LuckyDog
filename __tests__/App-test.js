jest.mock('react-native-gesture-handler');

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import App from '../src/App';

describe('App.js', () => {
  it('Should renders the app correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
