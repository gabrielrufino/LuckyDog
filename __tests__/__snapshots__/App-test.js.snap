// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js Should renders the app correctly 1`] = `null`;
